.github/workflows/label.yml

```diff
@@ -13,19 +13,31 @@ jobs:
       || github.event.label.name == 'E2-forcemacos'
       || github.event.label.name == 'A4-insubstantial'
       || github.event.label.name == 'A2-mergeoncegreen'
+
     steps:
       - uses: actions/checkout@v4
+
+      - uses: gear-tech/fork-action@v0.1.0
+        if: github.event.label.name != 'E2-forcemacos'
```

Comment:
This approach doesnt allow to run linux and macOS at the same time? :/

Reply:
https://github.com/gear-tech/gear/pull/3764/commits/b4ee445c9c43de3622bc4b207ff144fb86ef0001

Reply:
The labels here are triggered `async`, when we adding labels `A0-pleasereview` and `E2-forcemacos`, the dispatch workflow will be triggered for twice, once for `linux (A0)`, once for `macos (E2)`, see fc35380